### PR TITLE
python-exif: added package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Used for LinkScope tool to avoid the usage of Python venv: https://github.com/BlackArch/blackarch/issues/3659

Source AUR

### CODE REVIEW

File: packages/python-exif/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by Caleb Maclennan <caleb@alerque.com>
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:4]:
```suggestion
# This PKGBUILD was originally extracted from AUR.
# AUR Maintainer: Caleb Maclennan <caleb@alerque.com>
```

File: packages/python-exif/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+_py_name=exif
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:6]:
```suggestion
```

File: packages/python-exif/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+pkgname=python-$_py_name
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:7]:
```suggestion
pkgname=python-exif
_pkgname=${pkgname#python-}
```

File: packages/python-exif/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+pkgver=1.3.4
+pkgrel=1
+pkgdesc='Read and modify image EXIF metadata'
+arch=(any)
+url="https://gitlab.com/TNThieding/$_py_name"
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:12]:
```suggestion
url="https://gitlab.com/TNThieding/exif"
```

File: packages/python-exif/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# See COPYING for license details.
+
+# This file is based on the AUR package by Caleb Maclennan <caleb@alerque.com>
+
+_py_name=exif
+pkgname=python-$_py_name
+pkgver=1.3.4
+pkgrel=1
+pkgdesc='Read and modify image EXIF metadata'
+arch=(any)
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:11]:
```suggestion
arch=('any')
```

File: packages/python-exif/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+url="https://gitlab.com/TNThieding/$_py_name"
+license=(MIT)
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:13]:
```suggestion
license=('MIT')
```

File: packages/python-exif/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+_py_deps=(plum)
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:14]:
```suggestion
```

File: packages/python-exif/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+depends=(python
+         "${_py_deps[@]/#/python-}")
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:16]:
```suggestion
```

File: packages/python-exif/PKGBUILD [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+_py_name=exif
+pkgname=python-$_py_name
+pkgver=1.3.4
+pkgrel=1
+pkgdesc='Read and modify image EXIF metadata'
+arch=(any)
+url="https://gitlab.com/TNThieding/$_py_name"
+license=(MIT)
+_py_deps=(plum)
+depends=(python
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:15]:
```suggestion
depends=('python' 'python-plum')
```

File: packages/python-exif/PKGBUILD [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+         "${_py_deps[@]/#/python-}")
+_archive="$_py_name-$pkgver"
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:17]:
```suggestion
```

File: packages/python-exif/PKGBUILD [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+makedepends=(python-setuptools)
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:18]:
```suggestion
makedepends=('python-setuptools')
```

File: packages/python-exif/PKGBUILD [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+source=("https://files.pythonhosted.org/packages/source/${_py_name::1}/$_py_name/$_archive.tar.gz")
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:19]:
```suggestion
source=("https://files.pythonhosted.org/packages/source/${_pkgname::1}/$_pkgname/$_pkgname-$pkgver.tar.gz")
```

File: packages/python-exif/PKGBUILD [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+sha256sums=('abf3c125167a9ccc780e190966a84559b54627b3dfee04100753222753f5ce30')
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:20]: sha512

File: packages/python-exif/PKGBUILD [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+
+build() {
+	cd "$_archive"
+	python setup.py build
+}
+
+package() {
+	cd "$_archive"
+	python setup.py install --root="$pkgdir" --optimize=1 --skip-build
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:29]:
```suggestion
	python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
```

File: packages/python-exif/PKGBUILD [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+source=("https://files.pythonhosted.org/packages/source/${_py_name::1}/$_py_name/$_archive.tar.gz")
+sha256sums=('abf3c125167a9ccc780e190966a84559b54627b3dfee04100753222753f5ce30')
+
+build() {
+	cd "$_archive"
+	python setup.py build
+}
+
+package() {
+	cd "$_archive"
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:28]:
```suggestion
	cd "$_pkgname-$pkgver"
	
```

File: packages/python-exif/PKGBUILD [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+_py_deps=(plum)
+depends=(python
+         "${_py_deps[@]/#/python-}")
+_archive="$_py_name-$pkgver"
+makedepends=(python-setuptools)
+source=("https://files.pythonhosted.org/packages/source/${_py_name::1}/$_py_name/$_archive.tar.gz")
+sha256sums=('abf3c125167a9ccc780e190966a84559b54627b3dfee04100753222753f5ce30')
+
+build() {
+	cd "$_archive"
```
Comment (noraj) [inline packages/python-exif/PKGBUILD:23]:
```suggestion
	cd "$_pkgname-$pkgver"

```

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/python-exif/PKGBUILD (hunk 13 above)
Comment (D3vil0p3r) [reply packages/python-exif/PKGBUILD]: Done

**Pull request merged**